_boskos.yaml

```diff
@@ -0,0 +1,2951 @@
+# generated with generate-boskos.py; do not edit directly
```

Comment:
This file is added in error

Reply:
Thanks, I noticed it, I just changed the code